pkg/portfwd/client.go

```diff
@@ -130,14 +94,32 @@ func (g GrpcClientRW) Write(p []byte) (n int, err error) {
 func (g GrpcClientRW) Read(p []byte) (n int, err error) {
 	in, err := g.stream.Recv()
 	if err != nil {
-		if errors.Is(err, io.EOF) {
-			return 0, nil
-		}
 		return 0, err
 	}
-	if len(in.Data) == 0 {
-		return 0, nil
-	}
```

Comment:
This is not possible or just not needed, since the return value will be the same anyway if this happens?

Reply:
Not needed. We will not get empty data. Even if it happens technically we should simply write empty data to proxy conn as well. So removed it